diff -up abrt-2.12.0/configure.ac.omv~ abrt-2.12.0/configure.ac
--- abrt-2.12.0/configure.ac.omv~	2019-02-15 22:39:15.470508085 +0100
+++ abrt-2.12.0/configure.ac	2019-02-15 22:39:26.684690609 +0100
@@ -173,10 +173,10 @@ ABRT_PARSE_WITH([pythondoc]))
 [if test -z "$NO_PYTHONDOC"]
 [then]
     AM_CONDITIONAL(HAVE_PYTHON_SPHINX, true)
-    AC_PATH_PROG([PYTHON_SPHINX], [sphinx-build-3], [no])
+    AC_PATH_PROG([PYTHON_SPHINX], [sphinx-build], [no])
     [if test "$PYTHON_SPHINX" = "no"]
     [then]
-        [echo "The sphinx-build-3 program was not found in the search path. Please ensure"]
+        [echo "The sphinx-build program was not found in the search path. Please ensure"]
         [echo "that it is installed and its directory is included in the search path or"]
         [echo "pass --without-pythondoc to ./configure."]
         [echo "Then run configure again before attempting to build ABRT."]
