diff -up abrt-2.13.0/src/daemon/abrt-action-save-package-data.conf.omv~ abrt-2.13.0/src/daemon/abrt-action-save-package-data.conf
--- abrt-2.13.0/src/daemon/abrt-action-save-package-data.conf.omv~	2019-12-13 20:04:26.760375956 +0100
+++ abrt-2.13.0/src/daemon/abrt-action-save-package-data.conf	2019-12-13 20:04:43.079374887 +0100
@@ -7,3 +7,5 @@
 #
 # For details regarding the contents of this file and the defaults, see
 # `man 5 abrt-action-save-package-data.conf`.
+
+OpenGPGCheck = no
