diff --git a/src/daemon/abrt-action-save-package-data.conf b/src/daemon/abrt-action-save-package-data.conf
index 62c332a..8289a6c 100644
--- a/src/daemon/abrt-action-save-package-data.conf
+++ b/src/daemon/abrt-action-save-package-data.conf
@@ -7,7 +7,7 @@ OpenGPGCheck = no
 
 # Blacklisted packages
 #
-BlackList = nspluginwrapper,valgrind,strace
+BlackList = nspluginwrapper,valgrind,strace,nspluginwrapper-i386
 
 # Process crashes in executables which do not belong to any package?
 #
